src/server/rest/v1/service/AuthorizationService.ts

```diff
@@ -369,7 +369,7 @@ export default class AuthorizationService {
   }
 
   public static async checkAndGetTagAuthorizationFilters(tenant: Tenant, userToken: UserToken,
-      filteredRequest: Partial<Tag>, action: Action): Promise<AuthorizationFilter> {
+      filteredRequest: HttpByIDRequest | Partial<Tag>, action: Action): Promise<AuthorizationFilter> {
```

Comment:
`HttpByIDRequest` is used when checking a GET request & `Partial<Tag>` is used when checking a CREATE request.